job-descriptions/software-engineer-backend.md

```diff
@@ -0,0 +1,66 @@
+
+We value openness and transparency. [Our code](https://github.com/sourcegraph/sourcegraph), [our roadmap](https://about.sourcegraph.com/direction), and [our company processes](https://about.sourcegraph.com/company/open_source_open_company) are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+
+To create a product that serves the needs of all developers, we are building a diverse remote-first team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+As a backend engineer at Sourcegraph, you will focus on designing and developing fast and reliable services that power our product (e.g., code search, syntax highlighting, auth, code host integrations). Read our [project roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.g2wq4qci7wj0) to see the kinds of projects that we are working on today and have planned for the future.
+
+We are looking for candidates with the following skills and a strong desire to learn:
+
+- Excellent knowledge of Go.
+- Skilled at building, testing (e.g., unit, integration, end-to-end), and debugging high performance distributed systems in Go.
+- Good knowledge of Git in particular and other version control systems more generally.
+- Ability to communicate clearly and empathetically, especially in writing and documentation.
+- Practiced at creating high quality software balanced with a pragmatic understanding of how to make appropriate tradeoffs (e.g., cut scope) to ship quickly and iterate when necessary.
+
+Nice to have:
+
+- Experience making distributed systems observable (e.g., logging, Prometheus metrics, request tracing).
+- Experience using Postgres and Redis.
+- Experience load testing distributed systems.
+- Experience writing high performance Go code and benchmarks.
+- Published blog posts and/or tech talks about your work.
+- Experience working on high-performing teams, preferably tech startups.
+
+### Location
+
+We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world.
+
+### Compensation and benefits
+
+We provide [competitive pay and equity](https://about.sourcegraph.com/handbook/people-ops/compensation) because we want you to act like a business owner and share in the success of Sourcegraph. We also provide [many benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy, healthy, and productive.
```

Comment:
I see we are still missing a lot of information in the compensation section. This is quite an important part for people who look into remote work, since there are a lot of companies who treat remote workers as "cheap" work. We should make it clear we don't do that.

Reply:
Agree, the compensation part of the handbook is not fleshed out. I will propose some changes to the handbook to clarify the points you made.